style/haskell/README.md

```diff
@@ -5,13 +5,16 @@ Haskell
 
 * Break long expressions before operators or keywords.
 * Break long type signatures between arguments.
+* Don't use a blank line between pragma and module statements.
```

Comment:
Snoyman!!!! I hadn't gotten to his repos in the GitHub search I did. I just noticed that Snoyman did this to my github library, too. Bah. (Relatedly: GH is really bad at uniquing their search results.)

How about dropping this guide suggestion for now; we can revisit it after we write more code? The pragmas are distinct from the modules, afterall.